keyboards/handwired/ep40/readme.md

```diff
@@ -0,0 +1,15 @@
+# ep40
+
+![ep40](https://i.imgur.com/Jl7w6I9.jpg)
+
+A simple 40% keyboard
+
+Keyboard Maintainer: [Elliot Powell](https://reddit.com/u/e11i0t23)  
```

Comment:
Could you also put your github account here too, in addition to your reddit one.